src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -45,7 +51,7 @@ public override ExportResult Export(in Batch<Activity> batch)
                 this.WriteLine($"Activity.Duration:    {activity.Duration}");
                 if (activity.TagObjects.Any())
                 {
-                    this.WriteLine("Activity.TagObjects:");
+                    this.WriteLine("Activity.Tags:");
```

Comment:
Do you think folks might get confused here?

Reply:
TagObjects in an implementation detail only Export authors need to know right? Most end users use SetTag, so seeing "Tags" here as opposed to "TagObjects" looks nice in my view.